contrib/checkpoint/checkpoint-restore-cri-test.sh

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env bash
+
+set -eu -o pipefail
+
+DIR=$(dirname "${0}")
+
+cd "${DIR}"
+go build -o checkcriu
+
+if ! "./checkcriu"; then
+	echo "ERROR: CRIU check failed"
+	exit 1
+fi
+
+if [ ! -e "$(command -v crictl)" ]; then
+	echo "ERROR: crictl binary not found"
+	exit 1
+fi
+
+TESTDIR=$(mktemp -d -p "${PWD}")
+
+SUCCESS=0
+
+function cleanup() {
+	rm -f ./checkcriu
+	crictl ps -a || true
+	pkill containerd || true
+	if [ "${SUCCESS}" == "1" ]; then
+		echo PASS
+	else
+		echo "--> containerd logs"
+		sed 's/^/----> \t/' <"${TESTDIR}/containerd.log"
+		echo FAIL
+	fi
+	umount "$(find "${TESTDIR}" -name shm -type d | head -1)" >/dev/null 2>&1 || true
+	umount "$(find "${TESTDIR}" -name rootfs -type d | head -1)" >/dev/null 2>&1 || true
+	rm -rf "${TESTDIR}" || true
+}
+trap cleanup EXIT
+
+TESTDATA=testdata
+# shellcheck disable=SC2034
+export CONTAINERD_ADDRESS="$TESTDIR/c.sock"
+export CONTAINER_RUNTIME_ENDPOINT="unix:///${CONTAINERD_ADDRESS}"
+TEST_IMAGE=docker.io/library/alpine:latest
```

Comment:
`ghcr.io/containerd/alpine` (https://github.com/containerd/containerd/pkgs/container/alpine) might be better to avoid Docker Hub rate limit

Reply:
Good point. Changed.